test/unit/test-standard-actions.js

```diff
@@ -431,33 +427,29 @@ describes.sandboxed('StandardActions', {}, () => {
 
     it('should not add amp internal classes', () => {
       const element = createElement();
-      dummyInternalClasses.forEach(dummyInternalClass => {
-        const invocation = {
-          node: element,
-          satisfiesTrust: () => true,
-          args: {
-            'class': dummyInternalClass,
-          },
-        };
-        standardActions.handleToggleClass_(invocation);
-        expect(element.classList.contains(dummyInternalClass)).to.be.false;
-      });
+      const invocation = {
+        node: element,
+        satisfiesTrust: () => true,
+        args: {
+          'class': dummyInternalClass,
```

Comment:
Nit: Just inline the string here and below. Duplicating literals in tests is ok and preferred for readability.